app/models/friend_request.rb

```diff
@@ -0,0 +1,7 @@
+class FriendRequest < ApplicationRecord
+  validates :sender, presence: true
```

Comment:
No need to validate sender and receiver as your associations below take care of that for you